src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -482,19 +493,32 @@ private List<String> identifyWinners(
     } else {
       // We should only look for more winners if we haven't already filled all the seats.
       if (winnerToRound.size() < config.getNumberOfWinners()) {
-        // If the number of continuing candidates equals the number of seats to fill, everyone wins.
         if (currentRoundCandidateToTally.size()
             == config.getNumberOfWinners() - winnerToRound.size()) {
+          // If the number of continuing candidates equals the number of seats to fill,
+          // everyone wins.
           selectedWinners.addAll(currentRoundCandidateToTally.keySet());
+        } else if (config.isFirstRoundDeterminesThresholdEnabled()
+              && currentRoundCandidateToTally.size() - 1 == config.getNumberOfWinners()) {
+          // Edge case: if nobody meets the threshold, but we're on the penultimate round when
+          // isFirstRoundDeterminesThresholdEnabled is true, select the max vote getters as
+          // the winners.
+          BigDecimal maxVotes = currentRoundTallyToCandidates.lastKey();
+          selectedWinners = currentRoundTallyToCandidates.get(maxVotes);
         } else if (!config.isMultiSeatBottomsUpUntilNWinnersEnabled()) {
+          // Otherwise, select all winners above the threshold
           selectWinners(currentRoundTallyToCandidates, selectedWinners);
         }
       }
 
       // Edge case: if we've identified multiple winners in this round, but we're only supposed to
-      // elect one winner per round, pick the top vote-getter and defer the others to subsequent
-      // rounds.
-      if (config.isMultiSeatAllowOnlyOneWinnerPerRoundEnabled() && selectedWinners.size() > 1) {
+      // elect one winner per round, pick the top vote-getter. If this is a multi-winner election,
+      // defer the others to subsequent rounds. If this is a single-winner election where its
+      // possible  for no candidate to reach the threshold (i.e. "first round determines threshold"
+      // is set), the tiebreaker will choose the only winner.
+      boolean useTiebreakerIfNeeded = config.isMultiSeatAllowOnlyOneWinnerPerRoundEnabled()
+          || config.isFirstRoundDeterminesThresholdEnabled();
```

Comment:
What about the case in which it's single-winner and `isFirstRoundDeterminesThresholdEnabled` is false but we end up with two candidates who both have exactly half of the votes?

If that's not already handled, we should add a test for it, and I would recommend tweaking this to something like:
```
boolean tiebreakNeeded = selectedWinners.size() > 1
  && (config.getNumberOfWinners() == 1 || config.isMultiSeatAllowOnlyOneWinnerPerRoundEnabled());

Reply:
That would lead to one candidate being eliminated, so it wouldn't be handled here -- since nobody has reached the threshold, the tiebreaker runs at the elimination stage.

The test for this is in `tiebreak_seed_test`.

Reply:
Ah, so I think that test is wrong! It eliminates the tiebreak loser and runs a final round with only the winner continuing. I think we want to make that behavior consistent with what you've done here. @chughes297 can confirm.

To be specific, here's what that test is currently expecting:
```
{
  "config" : {
    "contest" : "Tiebreak test",
    "date" : "2017-12-03",
    "jurisdiction" : "Funkytown, USA",
    "office" : "Sergeant-at-Arms",
    "threshold" : "2"
  },
  "results" : [ {
    "round" : 1,
    "tally" : {
      "George Gervin" : "3",
      "Mookie Blaylock" : "3",
      "Yinka Dare" : "3"
    },
    "tallyResults" : [ {
      "eliminated" : "Yinka Dare",
      "transfers" : {
        "exhausted" : "3"
      }
    } ]
  }, {
    "round" : 2,
    "tally" : {
      "George Gervin" : "3",
      "Mookie Blaylock" : "3"
    },
    "tallyResults" : [ {
      "eliminated" : "George Gervin",
      "transfers" : {
        "exhausted" : "3"
      }
    } ]
  }, {
    "round" : 3,
    "tally" : {
      "Mookie Blaylock" : "3"
    },
    "tallyResults" : [ {
      "elected" : "Mookie Blaylock",
      "transfers" : { }
    } ]
  } ]
}
```

And I think what we actually want is for round 2 to be the final round, and in that round no one is eliminated but we elect Blaylock via the tiebreaker.

Reply:
> And I think what we actually want is for round 2 to be the final round, and in that round no one is eliminated but we elect Blaylock via the tiebreaker.

Yes!

Reply:
To keep this issue well-scoped, and since that's a bug that exists in `develop` today, I've created a separate issue to address it: https://github.com/BrightSpots/rcv/issues/694